bindings/python/iota_sdk/types/output_id.py

```diff
@@ -5,6 +5,14 @@
 
 
 class OutputId(dict):
+    """Represents an output ID.
+
+    Attributes:
+        output_id (HexStr): The unique id of an output.
+        transaction_id (HexStr): The transaction id associated with the output.
+        output_index (int): The index of the output within the transaction.
```

Comment:
types

Reply:
yeah ... that went into `part 2` (see previous commit message :stuck_out_tongue_closed_eyes: )